src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommandContext.cs

```diff
@@ -19,5 +19,13 @@ public class GetDocumentCommandContext
 
     public string ProjectName { get; set; }
 
+    // Can be OpenApi2_0, OpenApi3_0, etc.
```

Comment:
Is there a list of acceptable values somewhere?  I assume there's no `OpenApi2_1`, for example.  Are the names case-sensitive?

Reply:
Looks like the values come from `OpenApiSpecVersion`.  A cref might be helpful.